guides/common/attributes.adoc

```diff
@@ -126,7 +126,6 @@ ifdef::katello,foreman-el[]
 :PlanningDocURL: {BaseURL}Planning_Guide/index-foreman.html#
 :ProvisioningDocURL: {BaseURL}Provisioning_Guide/index-foreman.html#
 endif::[]
-
```

Comment:
Empty lines will make all those attributes not to be part of a header. I am not sure what it makes, looks like it works fine, but I would rather keep it the old way.

Reply:
If it must be that way to work, OK. But empty lines make this file more readable, the "blocks" more identifiable.